test/test_sparse.py

```diff
@@ -3338,25 +3338,21 @@ def assign_to():
 
         self.assertRaises(TypeError, assign_to)
 
-    def test_cpu_sparse_dense_mul(self, device):
-        # general multiplication is not supported, but 0dim multiplication is supported
+    def test_sparse_dense_mul(self, device):
         s = torch.sparse_coo_tensor([[0], [1]], [5.0], (2, 3), device=device)
```

Comment:
Since this PR handles also hybrid tensors, we should have the corresponding test case. For instance, use
```
        s_ = torch.sparse_coo_tensor([[0], [1]], [5.0], (2, 3), device=device)
        for s in [s_, s_.to_dense().to_sparse(1)]:
            t23 = s.to_dense()
            ...
```